City-of-Helsinki/helsinki-paatokset
# PP-195: Add upper section of agenda item

- Added new template for issues 
- Added styles
- Added new field top_category_name to paatokset_issues -migration

To test:

Run `drush updb`
Run paatokset_issues -migration
Check paatokset-issues entity display and see if it matches sketch
